daemon/src/common/String.h

```diff
@@ -241,6 +242,18 @@ namespace Str {
             return c;
     }
 
+    /*
+     * Converts a hexadecimal character to the value of the digit it represents.
+     * Pre: cisxdigit(ch)
+     */
+    inline CONSTEXPR_FUNCTION int GetHex(char ch)
```

Comment:
I made it into a constexpr because it used to be a macro in qcommon.
I guess it can become just a normal function, it's only being used in a couple places anyway.